digitaltwin_client/tests/dt_e2e/service/dt_e2e_test.ts

```diff
@@ -311,6 +312,58 @@ function veryAllInterfacesAreRegistered(done) {
     })
 }
 
+//
+// veryAllInterfacesAreRegistered makes sure that the e2e test interfaces were successfully registered.  These are stored
+// in properties of the modelDiscovery interface
+//
+function runTestCommandAndCheckMessageAtEventHub(testData:CommandTestData, numberInvocationAttempts:number, done) {
+    let receiveOK = false;
+    let ehClient: EventHubClient;
+
+    const startAfterTime = new Date(Date.now());
+    const onEventHubError = (err) => {
+      throw err;
+    };
+    const onEventHubMessage = (receivedMsg) => {
+        if((receivedMsg.annotations['iothub-connection-device-id'] === testDeviceInfo.deviceId) &&
+        (receivedMsg.annotations['iothub-interface-name'] === testData.interfaceName)){
+            console.log('received a message from the test device: ');
+            console.log(JSON.stringify(receivedMsg.body));
+            if (receivedMsg.body && receivedMsg.body === testData.payload) {
+                console.log('found telemetry message from test device. test successful.');
+                ehClient.close()
+                done()
+            } else {
+                console.log('Incoming device id is: ' + receivedMsg.annotations['iothub-connection-device-id']);
+            }
+        }
+    };
+
+    EventHubClient.createFromIotHubConnectionString(testHubConnectionString)
+        .then((client) => ehClient = client)
+        .then(() => ehClient.getPartitionIds())
+        .then((partitionIds) => {
+          partitionIds.forEach((partitionId) => {
+            ehClient.receive(partitionId, onEventHubMessage, onEventHubError, { eventPosition: EventPosition.fromEnqueuedTime(startAfterTime) });
+          });
+          return new Promise((resolve) => setTimeout(() => resolve(), 3000));
+        }).then(() => {
+            runTestCommand(testData, 0, (err, responsepPayload, responseStatus) => {
+                err = new Error('Faking error case')
```

Comment:
>err = new Error('Faking error case') [](start = 16, length = 36)

First could we have a better string than "Faking error case"?

I don't want to re-document TypeScript/Mocha in the comments of this test (since we don't anywhere else) but could you please let me know at least in comments or offline what's up with this line?  As a non-Node guy the intent here isn't clear.  We create an error object which should presumably always succeed (almost) and then done(err) when it does?

Reply:
I need to delete the line...  I was just force-testing error cases to see what is the behavior, and accidentally checked that in.  Thanks for catching that!!